front/app/templates/application.hbs

```diff
@@ -31,3 +28,5 @@
     {{outlet}}
 
 </div>
+<script> var $buoop = {vs:{i:11,f:-4,o:-4,s:9,c:-4},unsecure:true,mobile:false,api:4}; function $buo_f(){  var e = document.createElement("script");  e.src = "//browser-update.org/update.min.js";  document.body.appendChild(e);};try {document.addEventListener("DOMContentLoaded", $buo_f,false)}catch(e){window.attachEvent("onload", $buo_f)}</script>
+
```

Comment:
@DiscoStarslayer I just saw your comment but I think it's based on an outdated diff (I had a similar concern). What do you think about this solution using https://browser-update.org/ ?